### ISSUE TITLE
Didn't work for me

### ISSUE THREAD

**Issue by edalford11** [AlvaroFranco/AFDropdownNotification#2]
AFDropdownNotification *notification = [[AFDropdownNotification alloc] init];
notification.titleText = message;
notification.dismissOnTap = YES;
[notification presentInView:self.view withGravityAnimation:YES];

I don't need the buttons so I didn't set the delegate but it was not showing on my UITableViewController.

**Comment by KiranPanesar** [comment]
:+1: Seeing this too. Even when I use buttons and set a delegate…

**Comment by KiranPanesar** [comment]
Got it to work by making it a `strong` property.

**Comment by AlvaroFranco** [comment]
Yeah, looks like that does the trick. I'm currently looking into it. I'll keep you guys updated.

**Comment by animaonline** [comment]
It does´t seem to work in Swift, is there a reason for that?

_Update_
It started working when I defined the object inside the class, not the method.
